packages/core/src/checkbox/CheckboxIcon.css

```diff
@@ -68,6 +68,13 @@
 
 .saltCheckboxIcon-readOnly,
 .saltCheckbox:hover .saltCheckboxIcon-readOnly {
+  background: unset;
```

Comment:
This needs a change mention?

Reply:
Ah, unintentional. Didn't want to fix this yet